spec/javascripts/components/table-filter-spec.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env jasmine, jquery */
```

Comment:
I think you can remove `jquery` from this line as I didn't see any jQuery in this file.